pages/common/rails.md

```diff
@@ -0,0 +1,23 @@
+# rails
+
+> Ruby on Rails, or Rails, is a server-side MVC framework written in Ruby under the MIT License.
+
+- Check current version of rails:
```

Comment:
Make this as the last example.

Tip - Think of the natural order of usage that a user is going to do. First thing would be to create the project, start server and do other stuff. And then at last, there might be a need to check the version.